GoogleUtilities/Environment/SecureStorage/GULKeychainStorage.m

```diff
@@ -23,32 +23,33 @@
 #import "FBLPromises.h"
 #endif
 
-#import "FIRInstallationsErrorUtil.h"
-#import "FIRInstallationsKeychainUtils.h"
+#import <GoogleUtilities/GULSecureCoding.h>
 
-@interface FIRSecureStorage ()
+#import "GULKeychainUtils.h"
+
+@interface GULKeychainStorage ()
 @property(nonatomic, readonly) dispatch_queue_t keychainQueue;
 @property(nonatomic, readonly) dispatch_queue_t inMemoryCacheQueue;
 @property(nonatomic, readonly) NSString *service;
 @property(nonatomic, readonly) NSCache<NSString *, id<NSSecureCoding>> *inMemoryCache;
 @end
 
-@implementation FIRSecureStorage
+@implementation GULKeychainStorage
 
 - (instancetype)init {
   NSCache *cache = [[NSCache alloc] init];
   // Cache up to 5 installations.
   cache.countLimit = 5;
-  return [self initWithService:@"com.firebase.FIRInstallations.installations" cache:cache];
+  return [self initWithService:@"com.gul.KeychainStorage" cache:cache];
```

Comment:
Seems like the location has changed, how do we ensure the old keychain data is not wiped when update to this version? 
And do we care about backward compatibility?

Reply:
Good catch! Let me make sure the service name is alway set by client, so Installations will keep using the same service as before.